app/views/exams/_form.html.erb

```diff
@@ -0,0 +1,13 @@
+<%= simple_form_for(@exam) do |f| %>
+  <%= f.error_notification %>
+
+  <div class="form-inputs">
+    <%= f.input :score %>
```

Comment:
Oh hm I hadn't considered decimals! Let's put a ticket in the backlog to mask this field in the future.